src/genericsite/views.py

```diff
@@ -84,26 +57,39 @@ def get_object(self):
 
 
 class ArticleList(ListView):
+    """View for pages that present a list of articles (e.g. SectionPage, HomePage).
+
+    The `get_object` method is left unimplemented here, as it will be different for
+    each subclass. This is an extension of Django's views. In Django's GenericViews,
+    List pages don't have an `object`, but in GenericSite, all pages have an `object`.
+    """
+
     model = Article
-    paginate_by: int = 10
+    paginate_by: int = 15
```

Comment:
It might be nice to have this as a config option.

Reply:
Agreed, I'll add this as a requirement for the Pagination feature #12